plugins/@grouparoo/app-templates/src/source/table/getExamples.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  GetColumnDefinitionsMethod,
+  GetSampleRowsMethod,
+  DataResponseRow,
+  ColumnDefinitionMap,
+  FilterOperation,
+  SimpleAppOptions,
+} from "./pluginMethods";
+
+export interface GetExampleRowsMethod {
+  (argument: {
+    connection: any;
+    appOptions: SimpleAppOptions;
+    tableName: string;
+    columns?: ColumnDefinitionMap;
+    getSampleRows: GetSampleRowsMethod;
+    getColumns: GetColumnDefinitionsMethod;
```

Comment:
I think `getColumns` is optional (?) if columns is provided?

Reply:
that's true, but this is an internal method so I kept it simple. Not even sure how to type that.